pkg/lifecycle/render/helm/dependency.go

```diff
@@ -26,6 +26,10 @@ func (f *LocalTemplater) addDependencies(
 ) (depPaths []string, err error) {
 	for _, dependency := range dependencies {
 		if dependency.Repository != "" {
+			if strings.HasPrefix(dependency.Repository, "@") || strings.HasPrefix(dependency.Repository, "alias:") {
+				// The repository is an alias. Assume it has already been added.
+				return depPaths, nil
```

Comment:
Could we `continue` here instead, so that any other dependencies that we do have the ability to resolve get added?

Reply:
ahh, definitely. i misread the method signature. that should definitely `continue` rather than returning.

Reply:
fixed